### PR TITLE
Add Nightscout app

### PR DESCRIPTION
**Pull request by arvchristos**
Adding Nightscout Web portal: https://nightscout.github.io/

> Nightscout (also known as CGM in the Cloud) is an open-source cloud application used by people with diabetes and parents of kids with diabetes to visualize, store and share the data from their Continuous Glucose Monitoring sensors in real-time. Once setup, Nightscout acts as a central repository of blood glucose and insulin dosing/treatment data for a single person, allowing you to view the CGM graph and treatment data anywhere using just a web browser connected to the internet.

Implementation is based on the project `docker-compose.yml`: https://github.com/nightscout/cgm-remote-monitor/blob/master/docker-compose.yml

### CODE REVIEW

File: servapps/Nightscout/cosmos-compose.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      "{ServiceName}": {
+        "image": "nightscout/cgm-remote-monitor:latest",
+        "container_name": "{ServiceName}",
+        "restart": "unless-stopped",
+        "environment": [
+          "NODE_ENV=production",
+          "TZ=Etc/UTC",
+          "INSECURE_USE_HTTP=true",
+          "MONGO_CONNECTION=mongodb://{ServiceName}-mongo:27017/nightscout",
+          "API_SECRET=passwordpassword",
```
Comment (azukaar) [inline servapps/Nightscout/cosmos-compose.json:15]: can you use the password feature? {Passwords.0}

File: servapps/Nightscout/cosmos-compose.json [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    },
+    "minVersion": "0.4.0",
+    "services": {
+      "{ServiceName}": {
+        "image": "nightscout/cgm-remote-monitor:latest",
+        "container_name": "{ServiceName}",
+        "restart": "unless-stopped",
+        "environment": [
+          "NODE_ENV=production",
+          "TZ=Etc/UTC",
```
Comment (azukaar) [inline servapps/Nightscout/cosmos-compose.json:12]: TZ=auto

**Review by azukaar — CHANGES_REQUESTED** [review]

File: servapps/Nightscout/cosmos-compose.json (hunk 2 above)
Comment (arvchristos) [reply servapps/Nightscout/cosmos-compose.json]: Thank you! Fixed

File: servapps/Nightscout/cosmos-compose.json (hunk 1 above)
Comment (arvchristos) [reply servapps/Nightscout/cosmos-compose.json]: Thank you, fixed

By the way, is this `Passwords` feature documented somewhere? (please excuse my ignorance)

**Pull request merged**